ios/Sources/MediaFeature/SearchController.swift

```diff
@@ -0,0 +1,20 @@
+import UIKit
+
+@propertyWrapper
+class SearchController: NSObject {
+
+    let wrappedValue = UISearchController()
+    private let searchTextDidChangeTo: (String) -> Void
+
+    init(searchBarPlaceHolder: String? = nil, searchTextDidChangeTo: @escaping (String) -> Void) {
+        self.searchTextDidChangeTo = searchTextDidChangeTo
+        super.init()
+        wrappedValue.searchBar.delegate = self
+    }
+}
+
+extension SearchController: UISearchBarDelegate {
+    func searchBar(_ searchBar: UISearchBar, textDidChange searchText: String) {
+        searchTextDidChangeTo(searchText)
+    }
+}
```

Comment:
In my opinion, 
How about define this propertyWrapper in `Component` module ? 👀 

I know this is for `MediaFeature` only, but this `propertyWrapper` can be placed that place. 😄